### DISCUSSION TITLE
[Bug Report] Cannot install

### DISCUSSION THREAD

**Discussion by 10pointstogryffindor** [stashapp/stash#3351] [Q&A] [answered]
**Describe the bug**
Windows 10, I tried installing with the stash.exe file but keep failing to install with the following message: "Stash - Fatal Error Error: Panic: error initializing configuration: missing the following mandatory settings: generated".
I have no idea what's causing the installation to fail. Attached is the error message screenshot with the exact same message stated above.

![0102_010](https://user-images.githubusercontent.com/105363017/210242279-97731d57-4aae-418d-a8c0-b929c771c2d8.png)

**Comment by WithoutPants** [comment, marked as answer]
This occurs if the `generated` field is missing or empty in the `config.yml` file. This can occur in a failed install or if the config file was corrupted somehow. To resolve this, you can edit the `config.yml` file to add `generated: <path to generated directory>`, or (recommended for a new install) just delete the `config.yml` file. In your case, the `config.yml` file will be located in `C:\Users\<username>\.stash`.

**Reply by 10pointstogryffindor** [reply]
Yes, the issue has been resolved! Thank you very much.

**Comment by distkid** [comment]
Hey i have this problem except it started randomly, when i go to .stash, all i see is the "Plugins" folder. Never had any issues with stash previously, been using for months and database is still completely intact.
